lib/experimental/script-modules.php

```diff
@@ -200,3 +200,29 @@ function gutenberg_dequeue_module( $module_identifier ) {
 	_deprecated_function( __FUNCTION__, 'Gutenberg 17.6.0', 'wp_dequeue_script_module' );
 	wp_script_modules()->dequeue( $module_identifier );
 }
+
+/**
+ * Registers Gutenberg Script Modules.
+ *
+ * @since 19.3
+ */
+function gutenberg_register_script_modules() {
+	// When in production, use the plugin's version as the default asset version;
+	// else (for development or test) default to use the current time.
+	$default_version = defined( 'GUTENBERG_VERSION' ) && ! SCRIPT_DEBUG ? GUTENBERG_VERSION : time();
+
+	wp_register_script_module(
+		'@wordpress/a11y',
+		gutenberg_url( 'build-module/a11y/index.min.js' ),
+		array(),
+		$default_version
+	);
+	add_filter(
+		'script_module_data_@wordpress/a11y',
+		function ( $data ) {
+			$data['i18n'] = array( 'Notifications' => __( 'Notifications', 'gutenberg' ) );
```

Comment:
I'm not sure whether to use `'gutenberg'` here, the client doesn't.

```suggestion
			$data['i18n'] = array( 'Notifications' => __( 'Notifications' ) );
```